charts/dev/livebook/templates/common.yaml

```
@@ -1,2 +1,13 @@
 {{/* Render the templates */}}
-{{ include "tc.v1.common.loader.all" . }}
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render secrets for livebook */}}
+{{- $secrets := include "livebook.secrets" . | fromYaml -}}
+{{- if $secrets -}}
+  {{- $_ := set .Values.secret "secrets" $secrets -}}
```

```suggestion
  {{- $_ := set .Values.secret "livebook-secrets" $secrets -}}
```
This should match the $secretName

Now re-reading the cookie description, do you think we should do the same for that too?

---

So... if I "render" the chart with `helm template`, the secret is actually prefixed with `livebook-`. So in firefly, the secret is called `firefly-firefly-secrets`. 

So... what about this place?

---

> Now re-reading the cookie description, do you think we should do the same for that too?

Yup. just noticed that, too.

---

aaah in `_secrets.tpl` it was still wrong. I fixed it there, too.